code/modules/projectiles/projectile_base.dm

```diff
@@ -177,9 +177,10 @@
 			reagent_note += num2text(R.volume) + ") "
 		additional_log_text = "[additional_log_text] (containing [reagent_note])"
 
+	L.apply_effects(stun, weaken, knockdown, paralyze, irradiate, slur, stutter, eyeblur, drowsy, blocked, stamina, jitter)
```

Comment:
I'm a little wary of now just ignoring the value and not returning anything from this call, especially since quite a few overrides of `on_hit()` check if the parent call returned false, which will probably now always happen?